cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -230,7 +230,6 @@ if [ "${CURRENT_STORAGE}" = "etcd3" -a "${TARGET_STORAGE}" = "etcd2" ]; then
   rm -rf "${ROLLBACK_BACKUP_DIR}"
   mkdir -p "${ROLLBACK_BACKUP_DIR}"
   cp -r "${DATA_DIRECTORY}" "${ROLLBACK_BACKUP_DIR}"
-  rm -rf "${DATA_DIRECTORY}"/member/snap/*.snap
```

Comment:
@hongchaodeng - it was you who suggested it:
https://github.com/kubernetes/kubernetes/issues/36555#issuecomment-260224217